### ISSUE TITLE
Which assembler to use

### ISSUE THREAD

**Issue by gertk64** [rtrussell/BBCZ80#1]
Hi,

Which assembler do you use to assemble the source ?

Regards,
Gert

**Comment by rtrussell** [comment]
> Which assembler do you use to assemble the source ?

Usually Z80ASM (using the `/rmf` switch) but it will assemble with M80 too (using the `/z` switch).

**Comment by gertk64** [comment]
Thanks, it works with the supplied Z80ASM.COM and LINK.COM from the Windows zip file. I transferred these and the source files to a CPM disk image file for my (Teensy 4.0) Z80 emulator. I added some extra code in HOOKS.Z80 but found out if I add too much code and run over $4B00 the resulting BBCBASIC.COM crashes. 
In the MAKE.SUB file I see this address given as a parameter but I am not sure how it works. The linker says /p sets the program offset? Can I adjust this to a higher value?

**Comment by rtrussell** [comment]
> Can I adjust this to a higher value?

Yes, but it must be on a 'page boundary' (`xx00`).  If you look in [bin/acorn/MAKE.SUB](https://github.com/rtrussell/BBCZ80/blob/master/bin/acorn/MAKE.SUB) you'll see that it is set to `4C00` there.
